src/modules/gpu.c

```diff
@@ -44,12 +44,6 @@ void ffPrintGPU(FFinstance* instance)
     if(ffPrintFromCache(instance, FF_GPU_MODULE_NAME, &instance->config.gpu, FF_GPU_NUM_FORMAT_ARGS))
         return;
 
-    if(ffStrbufCompS(&ffDetectHost()->productName, FF_HOST_PRODUCT_NAME_WSL) == 0)
-    {
-        ffPrintError(instance, FF_GPU_MODULE_NAME, 0, &instance->config.gpu, "WSL doesn't expose senseful GPU names");
-        return;
-    }
-
```

Comment:
I don't see how GPU is detected on WSL other then pci, which will always expose the same name on all systems. Why delete this?

Reply:
Because that's what reported by the system. WSL2 is a virtual machine like vmware or virtualbox, which only provides virtual GPUs. As we don't detect specially for other VMs, we shouldn't detect WSL either. In addition, As WSL is continually improving, WSL may report the real GPU names some day. If a user don't like it, just disable the module with `--structure`

Note: we  can actually detect the real GPU names with [wmic.exe](https://learn.microsoft.com/en-us/windows/win32/wmisdk/wmic). However since we are running in WSL, we should honor what it reports.